integration/bridge.go

```diff
@@ -39,7 +40,7 @@ type bridge struct {
 
 func newBridge(addr string) (*bridge, error) {
 	b := &bridge{
-		inaddr:  addr + ".bridge",
+		inaddr:  fmt.Sprint(addr, rand.Intn(30000)),
```

Comment:
This doesn't help with still collisions:
port=1, pid=1, rand=10 => 1 000001 10
port=10, pid=11, rand=0 => 10 000011 0

I think the 0 is better since it's not introducing more changing parts. Needs some commenting about the schema needing port numbers, though:
`// bridge "port" is ("%05d%05d0", port, pid) since go1.8 expects the port to be a number`